telco-examples/edge-clusters/aarch64/eib/custom/files/performance-settings.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# Script generate to tuning the performance of the system for running Telco Workloads
+# This script is intended to be run on a worker node in a Telco Edge Cluster
+
+if [ "`whoami`" != "root" ]; then
+        echo root required to quiten machine
```

Comment:
Not sure what `quiten` means here. Could you explain?

Reply:
good catch! changed sorry